bsc.evaluator/undefined/undefined.exp

```diff
@@ -10,6 +10,8 @@ if {$ctest == 1} {
   compare_file AbstractDeriveType.bsv.bsc-ccomp-out
   compile_object_fail_error AbstractDerivePosition.bsv S0015
   compare_file AbstractDerivePosition.bsv.bsc-ccomp-out
+  compile_object_fail_error AbstractDeriveAttrib.bs S0015 1 sysAbstractDeriveAttrib
```

Comment:
I would assume that you can leave off the arguments after `S0015` (as in the preceeding tests)

Reply:
The test fails to give an error when I leave those off, it doesn't seem to be running the evaluator?  Could this be an issue with .bs vs. .bsv in the tcl test commands?  I looked at it a bit and didn't notice anything obvious, though.